pkg/virt-operator/resource/apply/crds.go

```diff
@@ -113,8 +113,6 @@ func (r *Reconciler) createOrUpdateCrd(crd *extv1.CustomResourceDefinition) erro
 			return err
 		}
 		log.Log.V(2).Infof("crd %v updated", crd.GetName())
-	} else {
-		log.Log.V(2).Infof("crd %v is up-to-date", crd.GetName())
```

Comment:
lets instead change this to log level 4 to match other resources

Reply:
updated, thanks!